spatialpandas/io/parquet.py

```diff
@@ -28,6 +28,7 @@
     _maybe_prepend_protocol,
     validate_coerce_filesystem,
 )
+from .. import PANDAS_GE_12
```

Comment:
I was thinking this might be needed in multiple modules, but it looks like it is only used here.  If that's the case, I think we should just move the version check into here rather than in `__init__.py` as I previously suggested.

Reply:
I'm fine with either. If we end up needing this in more places then I agree we should move it to the `__init__.py`.